src/payment/strategies/stripev3/stripev3-payment-strategy.spec.ts

```diff
@@ -24,15 +26,20 @@ import {
     PaymentMethodRequestSender,
     PaymentRequestSender
 } from '../../../payment';
-import { PaymentArgumentInvalidError } from '../../errors';
+import { getShippingAddress } from '../../../shipping/shipping-addresses.mock';
+import {PaymentArgumentInvalidError} from '../../errors';
```

Comment:
Add spaces between the curly bracket.
```suggestion
import { PaymentArgumentInvalidError } from '../../errors';
```